Libraries/Network/XMLHttpRequest.android.js

```diff
@@ -26,6 +26,11 @@ function convertHeadersMapToArray(headers: Object): Array<Header> {
 }
 
 class XMLHttpRequest extends XMLHttpRequestBase {
+  constructor() {
+    super();
+    // android supports upload
+    this.upload = {};
```

Comment:
we should move this to `XMLHttpRequestBase` as both Android and iOS support upload